packages/patternlab/styleguide/source/_patterns/01-atoms/11-text/link.json

```diff
@@ -4,6 +4,7 @@
     "text":"This is a link",
     "labelContext": "to show as an example",
     "info": "",
+    "srLabel": "",

```

Comment:
This is a nitpicking comment — can we rename the `srLabel` var name to `ariaLabel` instead? I see we use `ariaLabel` in other places

Reply:
Changed it as suggested.